packages/desktop-client/src/components/reports/graphs/adjustTextSize.ts

```diff
@@ -0,0 +1,88 @@
+export const adjustTextSize = (
+  sized: number,
+  type: string,
+  width?: number,
+): `${number}px` => {
+  let source;
+  switch (type) {
+    case 'variable':
+      const findArray = variableLookup.find(({ value }) => width < value).arr;
```

Comment:
I feel like there's a better way to do this. My brain just can't come up with anything better right now. Thoughts?